.github/workflows/tag.yml

```
@@ -14,24 +14,30 @@ jobs:
       packages: write
 
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
       - name: Docker meta Service Name for docker hub
         id: docker_meta_img_hub
         uses: docker/metadata-action@v4
         with:
-          images: schulcloud/version-aggregator
+          images: docker.io/schulcloud/version-aggregator, quay.io/schulcloud/schulcloud-server
```

Copy-paste mistake, the quay repo refers to `schulcloud-server` instead of `version-aggregator`
```suggestion
          images: docker.io/schulcloud/version-aggregator, quay.io/schulcloud/version-aggregator
```